src/classes/CW_AudienceCSVProcessController.cls

```diff
@@ -0,0 +1,131 @@
+/**
+* File Name   :    CW_AudienceCSVProcessController.cls
+* Description :    Page Contrloer for second step in importing Audience Rewards
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex Panka	  Intial 
+*/
+
+public with sharing class CW_AudienceCSVProcessController {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.